js/index.js

```diff
@@ -185,28 +184,28 @@ $(document).ready(function(){
 			updateSmsList();
 			setInterval(updateSmsList, 5000);
 		});
-  
+
 		$('#smss').delegate('tr', 'click', function(e) {
 			e.preventDefault();
 
 			var smsId = $(this).closest('tr').attr('id');
-			
+
 			$('.chat-with').empty();
 			$(this).closest('tr').removeClass("unread");
-			
+
 			var txt = $(this).closest('tr').children('td.from').text()
 			$('.chat-with').text('Chat with ' + txt);
 			$('#sms-to-phone').val('+' + smsId);
 			$('#sms-from-phone').val($(this).closest('tr').children('td.to').text().substr(3));
-			
+
 			getSmsHistory('+' + smsId);
 		});
 
 		$('#sendSMS').click(function(e) {
 			e.preventDefault();
-			
-			$.post( "/messages/sms/" + $('#sms-messageid').val(),{to:$('#sms-to-phone').val(),from:$('#sms-from-phone').val(),content:$('#message-to-send').val()}, function( data ) {
-				
+
+			$.post( OpenVBX.home + "/messages/sms/" + $('#sms-messageid').val(),{to:$('#sms-to-phone').val(),from:$('#sms-from-phone').val(),content:$('#message-to-send').val()}, function( data ) {
```

Comment:
Someone else just reached out to me having an issue with sending sms message from within the plugin.  When I realized they had installed OpenVBX in a subfolder I remember you had already solved the problem.  Thanks for the commit.